src/components/Crowdsale/CrowdsaleSummary.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+export const CrowdsaleSummary = ({
+  contributorsCount,
+  currentRatePerETH,
+  endRatePerETH,
+  extraClassName = '',
+  isMintedCappedCrowdsale,
+  startRatePerETH,
+  tokensClaimed,
+  totalSupply
+}) => {
+  const isDutchAuctionCrowdsale = !isMintedCappedCrowdsale
```

Comment:
`!mintedCapped` isn't necessarily `dutchAuction`. Currently it is, but if new strategies will be added in the future we can just make it a bit more generic like passing `strategies` in the props. What do you think?

```suggestion
  const { isMintedCappedCrowdsale, isDutchAuction } = strategies
```

and in the `index.js` file do something like:

```
const strategies = { isMintedCappedCrowdsale, isDutchAuction }

// ...
<CrowdsaleSummary
  {...}
  strategies={strategies}
  {...}
/>
```

Reply:
Yeah, you are right, but I did something different anyway: I created a component for each strategy (_CrowdsaleSummaryMintedCapped_ and _CrowdsaleSummaryDutchAuction_), to make testing and programming a little easier.